### PR TITLE
build foreman for focal

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: debian/focal/foreman/control [hunk 1]
```diff
@@ -0,0 +1,229 @@
+Source: foreman
+Maintainer: Greg Sutcliffe <greg.sutcliffe@gmail.com>
+Section: web
+Priority: extra
+Standards-Version: 3.9.1
+Build-Depends: debhelper (>= 9.20160709), bundler (>= 1.3), ruby2.5-dev, git | git-core,
```
Comment (evgeni) [inline debian/focal/foreman/control:6]: heh…

**Review by ekohl — COMMENTED** [review]
I think it makes sense to have a Redmine issue for the unversioned Ruby use so it's tracked in the changelog. Then I'm ok with it.

**Review by ekohl — APPROVED** [review]

**Comment by evgeni** [comment]
If only it would build…

```
12:09:35 /usr/bin/bundle exec rake locale:pack RAILS_ENV=production
12:09:54 rake aborted!
12:09:54 TypeError: superclass mismatch for class Command
12:09:54 /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:2:in `<class:Thor>'
```

**Comment by ekohl** [comment]
That does sound vaguely familiar. The internet shows https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=955405 and https://github.com/erikhuda/thor/issues/721

**Comment by evgeni** [comment]
Yes, that's also what I have open right now.

I think the easiest path for us would be to build thor 1.0.1 or 1.1.0 as a package, so that it matches what we vendor

**Comment by ekohl** [comment]
https://bugs.launchpad.net/ubuntu/+source/ruby-thor/+bug/1885424 suggests that too so :+1: for that.

**Comment by evgeni** [comment]
[test packaging]

**Comment by evgeni** [comment]
nope, still `TypeError: superclass mismatch for class Command`

**Comment by evgeni** [comment]
[test packaging]

**Comment by evgeni** [comment]
This is finally green (or will be once the other targets finish)

**Review by ekohl — APPROVED** [review]

**Pull request merged**